### PR TITLE
PR for wrong code scripts in projections.rst

### PR DESCRIPTION
**Pull request by mzl9039**
hi, rougier, I make this pr for the [issue](https://github.com/rougier/scientific-visualization-book/issues/34). But the vscode seemed to have deleted some blank spaces automatically, although the blank line keeps, i'm not sure if this will impact during compiling.

### CODE REVIEW

File: rst/projections.rst [hunk 1]
```diff
@@ -74,31 +74,31 @@ distances are the same:
...
-   >>> print( (transform(P1)-transform(P0))[0] )
+   >>> print( (transform(P3)-transform(P2))[0] )
    155.0
 
 If your data has negative values, you have to use a symmetric log scale (symlog_)
 that is a composition of both a linear and a logarithmic scale. More precisely,
 values around 0 use a linear scale and values outside the vicinity of zero uses
 a logarithmic scale. You can of course specify the extent of the linear zone
-when you set the scale. The logit_ scale is used for values in the range ]0,1[
+when you set the scale. The logit_ scale is used for values in the range [0,1]
```
Comment (rougier) [inline rst/projections.rst:84]: Why this change?

Comment (mzl9039) [reply rst/projections.rst:84]: Sorry for the late response! Maybe I should use 2 commits here. Change the square brackets because they are wrong because we use the left bracket before the range and the right bracket after the range, namely "[0, 1]", not "]0, 1[".

Comment (mzl9039) [reply rst/projections.rst]: OMG, I'm really sorry, I'll update the change today!

Comment (rougier) [reply rst/projections.rst]: No problem. I se you made another PR, I'll merge the new one.
